crates/nu-protocol/src/hir.rs

```diff
@@ -656,7 +656,7 @@ impl SpannedExpression {
                 // Higher precedence binds tighter
 
                 match operator {
-                    Operator::Multiply | Operator::Divide | Operator::Modulo => 100,
+                    Operator::Multiply | Operator::Divide | Operator::Modulo | Operator::Pow => 100,
```

Comment:
https://en.wikipedia.org/wiki/Order_of_operations

Exponents are higher precedence than multiple and divide

Reply:
made pow 100 and the others lower.